src/interfaces/date-interfaces.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * ISO date string (YYYY-MM-DD), assumed to be in Pacific timezone.
+ * @example "2021-12-31"
+ */
+ export type IsoDatePacific = string
```

Comment:
This is so we can declare a variable of this type and know what format it should be.